controllers/reclaimspacejob_controller.go

```diff
@@ -18,45 +18,332 @@ package controllers
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"math"
+	"time"
 
+	csiaddonsv1alpha1 "github.com/csi-addons/kubernetes-csi-addons/api/v1alpha1"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/connection"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/proto"
+	"github.com/csi-addons/spec/lib/go/identity"
+
+	"github.com/go-logr/logr"
```

Comment:
with all the kubernetes packages already, I wonder why klog is not used :thinking:

Reply:
https://github.com/go-logr/logr 
Logr is just a common logging interface used commonly.